### ISSUE TITLE
Installed but not working.

### ISSUE THREAD

**Issue by marlon-shiftone** [codemapweb/codemap_extension#32]
I have installed the extension via SV code extensions but it is not working. I have created empty files .json and .c as described but it is not working. The link [[How to generate them]] is broken. Is it still having maintenance?

Thanks

**Comment by megawarpx** [comment]
Yes, even now the links don't work. There is not even the "test" folder mentioned in the instructions. It seems like an abandoned project. :'(
